apps/core/scenes/bookingDetail/menuGroups/Passengers/PassengersList.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import Passenger from './Passenger';
+import type { PassengersList as PassengersType } from './__generated__/PassengersList.graphql';
+
+type Props = {|
+  +data: ?PassengersType,
+|};
+
+const renderPassenger = passenger =>
+  passenger && (
+    <Passenger
+      key={passenger.id}
+      title={passenger.title}
+      firstname={passenger.firstname}
+      lastname={passenger.lastname}
+      birthday={passenger.birthday}
+    />
+  );
+
+const PassengersList = (props: Props) => {
+  const passengers = props.data?.passengers;
+  return (
+    <View style={styles.container}>
+      {passengers && passengers.map(renderPassenger)}
+    </View>
+  );
+};
+
+export default createFragmentContainer(
+  PassengersList,
+  graphql`
+    fragment PassengersList on BookingInterface {
+      passengers {
+        id
+        title
+        firstname
+        lastname
+        birthday
```

Comment:
You are not using any of these values in this component. Please create a fragment out of the passenger component.

Reply:
Just created Passenger fragment